build/server-image/kubectl/Dockerfile

```diff
@@ -14,10 +14,21 @@
 
 # Dockerfile used for the kubectl image.
 
-ARG BASEIMAGE
 ARG BINARY
 
 
-FROM "${BASEIMAGE}"
+FROM registry.k8s.io/build-image/debian-base:bookworm-v1.0.0
```

Comment:
@rayandas Can you explain why do we change base image? Can we simply install packages that we need?

Reply:
The default image is distroless hence it doesn’t have a shell. So I used the debian base image suggested by @xmudrii 
Please go through his [comment above](https://github.com/kubernetes/kubernetes/pull/119592#issuecomment-1651945816). 
Thanks.

Reply:
Thanks for the explanations. Now it makes sense to me.